tile-json/README.md

```diff
@@ -0,0 +1,31 @@
+# Display all stations with the Vector Tile Server
+
+This tutorial explains how to show charging stations on a map by using the GeoJSON response from Chargetrip Vector Tile Server:
```

Comment:
This tutorial guides you through the process of showing charging stations on a map by using the GeoJSON response from the Chargetrip Tile Service.